src/components/activity-groups-list/activity-group-item/activity-group-details.tsx

```diff
@@ -0,0 +1,129 @@
+import { useCollectibleBySlugSelector } from 'src/store/wallet/wallet-selectors';
+import { formatSize } from 'src/styles/format-size';
+import { isDefined } from 'src/utils/is-defined';
+import { tzktUrl } from 'src/utils/linking.util';
+
+import { ActivityGroupAmountChange, TextSize } from './activity-group-amount-change/activity-group-amount-change';
+import { ActivityGroupDollarAmountChange } from './activity-group-dollar-amount-change/activity-group-dollar-amount-change';
+import { useActivityGroupItemStyles } from './activity-group-item.styles';
+import { BAKING_REWARDS_TEXT } from './activity-group-type/use-activity-group-info.hook';
+import { ActivityStatusBadge } from './activity-status-badge/activity-status-badge';
+import { ActivityTime } from './activity-time/activity-time';
+import { ActivityGroupItemSelectors } from './selectors';
+
+interface ActivityDetailsProps {
+  activity: ActivityInterface;
+  nonZeroAmounts: ActivityNonZeroAmounts;
+  label: string;
+  address: string | undefined;
+  transactionHash: string;
+  transactionType: string;
+  transactionSubtype: string;
+}
+
+export const ActivityDetails: FC<ActivityDetailsProps> = ({
+  activity,
+  nonZeroAmounts,
+  label,
+  address,
+  transactionHash,
+  transactionType,
+  transactionSubtype
+}) => {
+  const styles = useActivityGroupItemStyles();
+  const selectedRpcUrl = useSelectedRpcUrlSelector();
+
+  const [areDetailsVisible, setAreDetailsVisible] = useState(false);
+  const handleOpenActivityDetailsPress = useCallback(() => setAreDetailsVisible(prevState => !prevState), []);
+
+  const collectible = useCollectibleBySlugSelector(`${activity.address}_${activity.tokenId ?? 0}`);
+  const isCollectible = useMemo(() => isDefined(collectible), [collectible]);
+
+  return (
+    <View>
+      <View style={[styles.row, styles.justifyBetween]}>
+        <View style={[styles.row, styles.itemsCenter]}>
+          <ActivityStatusBadge status={activity.status} />
+          <Divider size={formatSize(4)} />
+          <ActivityTime timestamp={activity.timestamp} />
+        </View>
+        <TouchableWithAnalytics testID={ActivityGroupItemSelectors.details} onPress={handleOpenActivityDetailsPress}>
+          <Icon name={areDetailsVisible ? IconNameEnum.DetailsArrowUp : IconNameEnum.DetailsArrowDown} />
+        </TouchableWithAnalytics>
+      </View>
+      {areDetailsVisible && (
+        <View style={styles.card}>
+          {!isEmpty(nonZeroAmounts.amounts) && (
+            <View style={[styles.detailItem, styles.detailItemBorder, styles.itemsStart]}>
+              <Text style={styles.detailText}>{transactionSubtype}</Text>
+              <View style={styles.row}>
+                <View style={styles.itemsEnd}>
+                  <ActivityGroupAmountChange nonZeroAmounts={nonZeroAmounts} textSize={TextSize.Small} />
+                  {isCollectible ? (
+                    <Text style={styles.ntfPrice}>0.00$</Text>
```

Comment:
Please use `useCurrentFiatCurrencyMetadataSelector` to get the actual symbol to be used